api_fhir_r4/defaultConfig.py

```diff
@@ -5,6 +5,7 @@
         "female": "F",
         "other": "O"
     },
+    "host_domain": "http://127.0.0.1:8000",
```

Comment:
Try to get the host domain from "self.request.build_absolute_uri()". Thanks.

Reply:
ok